edge/test/integration/utils/helpers/helpers.go

```diff
@@ -349,18 +350,20 @@ func GetPods(EdgedEndpoint string) (v1.PodList, error) {
 
 //CheckPodRunningState is function to check the Pod state
 func CheckPodRunningState(EdgedEndPoint, podname string) {
-	gomega.Eventually(func() string {
-		var status string
+	gomega.Eventually(func() error {
 		pods, _ := GetPods(EdgedEndPoint)
 		for index := range pods.Items {
 			pod := &pods.Items[index]
 			if podname == pod.Name {
-				status = string(pod.Status.Phase)
 				common.Infof("PodName: %s PodStatus: %s", pod.Name, pod.Status.Phase)
+				if pod.Status.Phase == v1.PodRunning || pod.Status.Phase == v1.PodSucceeded {
```

Comment:
For redis and nginx, `PodSucceeded` is the wrong Status, so close this first.
/close